src/app/modules/item/pages/item-log-view/item-log-view.component.html

```diff
@@ -47,18 +47,27 @@
           <td *ngFor="let col of columns">
             <ng-container [ngSwitch]="col.field">
               <ng-container *ngSwitchCase="'activityType'">
-                {{ rowData.activityType | logActionDisplay : rowData.score }}
-                <ng-container *ngIf="rowData.displayLoadButton">
-                  &nbsp;(
-                  <a
-                    class="alg-link"
+                <div class="activity-container">
+                  <div class="activity-left">
+                    <span class="activity-progress">
+                      <alg-score-ring
+                        [currentScore]="rowData.score"
+                        [diameter]="32"
+                        *ngIf="rowData.score"
+                      ></alg-score-ring>
+                    </span>
+
+                    {{ rowData.activityType | logActionDisplay }}
+                  </div>
+
+                  <button
+                    pButton
+                    class="alg-button p-button-rounded p-small-button"
+                    i18n-label="text on button to load a submission" label="Load"
```

Comment:
Could you label this button "Reload answer" when not observing, and "View answer" when observing ? 
You can add the observing-or-not information in item-log-component.ts at line 75.

Reply:
Fixed